container_workflow_tool/main.py

```diff
@@ -117,6 +117,8 @@ def _setup_args(self, args):
             self.set_commit_msg(args.commit_msg)
         if getattr(args, 'rebuild_reason', None) is not None and args.rebuild_reason:
             self.rebuild_reason = args.rebuild_reason
+        if getattr(args, 'gitlab', None) is not None and args.gitlab:
```

Comment:
First of all, these two lines seem to be here twice, see lines 130 and 131.

And IMHO, if an argument uses `store_true` or `store_false` actions, it has to be True or False, nothing else, which means that the check for the attribute presence and the default None here are not needed. The argument will always be there.

Reply:
Some options are valid only for a specific subcommand, like gitlab. But I will check what can be enhanced. You are right.